CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ New features:
 - `spago install purescript-XYZ` will now strip `purescript-` prefix and install XYZ (if it exists in package set) instead of just failing with a warning (#367)
 - `spago run` now recognizes backend specified in the configuration file and calls the backend with `--run` argument.
 - documentation now includes a step-by-step guide on setting up a Spago/Parcel project (#456)
+- `spago output-path` returns the output path so that it can be shared with tools such as `purs-loader`
```

Comment:
There are still 5 occurrences of `output-path` in this PR, that should be changed to `path`.